# Aufgabenblatt 1

## Kombinatorik

### Aufgabe 1

Wie viele gleich lange Worte lassen sich aus dem Wort STATISTIK bilden?


In der ersten Aufgabe wird gesucht auf wieviele verschiedene Arten man aus dem Wort "STATISTIK" gleich lange Worte erhält. Jede der möglichen Permutationen muss aus den selben Buchstaben bestehen welche im Wort STATISTIK vorkommen.
Abstraktiv betrachtet kann man sich das Wort "STATISTIK" als eine n-elementige Menge vorstellen, wobei jedes der vorkommenden Buchstaben im Wort ein Element ist. Würde jeder Buchstabe nur einmal vorkommen, so wäre die Lösung einfach nur die Faktorielle von n.

Das Wort "STATISTIK" besitzt 5 verschiedene Elemente: S, T, A, I, K Insgesamt besitzt das Wort STATISTIK aber 9 Worte. Wäre jedes der 9 Worte ein anderer Buchstabe dann wäre die Lösung der Aufgabe nur die Faktorielle von 9.

S kommt im Wort 2 mal vor (2-elementige Teilmenge), T kommt im Wort 3 mal vor (3-elementige Teilmenge und I kommt im Wort 2 mal vor (2-elementige Teilmenge).

Für die Lösung muss daher die Faktorielle von 9 muss durch die Faktoriellen der mehrmals vorkommenden Buchstaben dividiert werden (bei der 3-elementigen Teilmenge von den 3 mal vorkommenden T ist es egal auf welcher Stelle T ist)


In [1]:
x <- factorial(9)/(factorial(2) * factorial(3) * factorial(2))
x

[1] 15120

$$\Large \frac{9!}{2! \,\, \cdot \,\, 3! \,\, \cdot \,\, 2! \,\,} \,\, = \,\, 15120 $$

### Aufgabe 2

Einer Studentin werden bei einer Prüfung 12 Fragen vorgelegt. Sie darf davon 4 auswählen. Man berechne die Anzahl der Wahlmöglichkeiten, wenn sie aus den ersten 6 Fragen mindestens drei auswählen muss.

Für diese Aufgabe wird der Binomialkoeffizient benötigt. Der Binomialkoeffizient gibt die Anzahl der Möglichkeiten an wie man aus einer n-elementigen Menge eine k-elementige Teilmenge ziehen kann.
$$\left( \begin{array}{c} n \\ k \end{array}\right) \,\, = \,\, \frac{n!}{(n-k)! \,\, \cdot \,\, k!}$$

Die Studentin soll darf aus allen 12 Fragen genau 4 auswählen. Aus den ersten 6 Fragen soll sie mindestens 3 Fragen auswählen. Sie 2 Möglichkeiten.
Sie kann aus den ersten 6 Fragen genau 3 Fragen auswählen und aus den anschließenden 6 Fragen genau eine auszuwählen. Wir berechnen dazu also die Anzahl der Arten eine 3-elementigen Teilmengen welche aus einer 6 elementigen Menge zu ziehen und die Anzahl der Arten eine 1-elementige Teilmenge aus einer 6 elementigen Menge zu ziehen:
$$\Large \left( \begin{array}{c} 6 \\ 3 \end{array}\right) \,\, \cdot \,\, \left( \begin{array}{c} 6 \\ 1 \end{array}\right) \,\, = \,\, 120 $$

In [7]:
x <- choose(6,3) * choose(6,1)
x

[1] 120

Die andere Möglichkeit ist für die Studentin alle 4 Fragen aus den ersten 6 Fragen auszuwählen und keine Frage aus den anschließenden 6 Fragen auszuwählen. Die Anzahl der Möglichkeiten berechnet sich dann genau wie oben.
$$\Large \left( \begin{array}{c} 6 \\ 4 \end{array}\right) \,\, \cdot \,\, \left( \begin{array}{c} 6 \\ 0 \end{array}\right) \,\, = \,\, 15 $$

In [6]:
x <- choose(6,4) * choose(6,0)
x

[1] 15

Um die Gesamtanzahl aller Möglichkeiten zu erhalten werden die Anzahl der Möglichkeiten für beide Fälle addiert.
$$\Large 120 \,\, + \,\, 15 \,\, = \,\, 135 \,\,$$

Die Studentin hat also unter den gegebenen Vorraussetzungen genau 135 Möglichkeiten wie sie die 4 Fragen aus den 12 Fragen auswählen kann.

### Aufgabe 3

Bei einem Radrennen mit 10 Startern wird in jeweils 4 Rennen der
Sieger ermittelt. Wie viele mögliche Gesamtergebnisse gibt es, wenn
nur die Anzahl der gewonnenen Rennen berücksichtigt wird?

Pro Rennen kann nur ein Fahrer gewinnen. Nachdem ein Fahrer gewonnen hat, kann er aber beim nächsten Rennen wieder gewinnen (Ziehung mit Zurücklegen).
Hier ist auch nicht wichtig ob ein Fahrer im 1., 2., 3. oder 4. Rennen gewinnt. Daher muss die folgende Formel verwendet werden (ungeordnete Ziehung).

$$\Large \left( \begin{array}{c} n + k - 1\\ k \end{array}\right) \,\, = \,\, \frac{(n +k -1)!}{(n - 1)! \,\, \cdot \,\, k!}$$

Es gibt insgestamt 10 Starter (n = 10) und 4 Rennen (k = 4). Die Ziehung eines Starters ist gleichzusetzen mit dem Gewinnen eines Fahrers.

$$\Large \left( \begin{array}{c} 10 + 4 - 1\\ 4 \end{array}\right) \,\, = \,\, 715 \,\,$$

Es gibt daher insgesamt 715 Möglichkeiten wie die 10 Radfahrer eines der 4 Rennen gewinnen können.

In [21]:
x <- choose(13,4)
x

[1] 715

### Aufgabe 4

Aus 5 Ehepaaren werden zufällig 4 Personen ausgewählt. Wie viele unterschiedliche Anordnungen gibt es, wenn sich unter den 4 ausgewählten Personen kein Ehepaar befinden soll?

Ingesamt gibt es genau 10 Personen weil jedes Ehepaar aus 2 Personen besteht. Es gibt 10 Möglichkeiten wie man eine Person von 10 wählen kann. Das würde dem Binomialkoeffizient $ \left( \begin{array}{c} 10\\ 1 \end{array}\right) \,\, = \,\, 10 \,\,$ entsprechen.

Für die 2. Person gibt es 8 Möglichkeiten (Binomialkoeffizient $ \left( \begin{array}{c} 8\\ 1 \end{array}\right)$) diese zu ziehen. Die 2. Person darf nicht der Ehemann/die Ehefrau der 1. gezogenen Person sein. Deswegen nurnoch 8 Möglichkeiten statt 9.
Für die 3. Person gibt es nur noch 6 Möglichkeiten und für die 4. Person 4 Möglichkeiten.

Daher lautet die Rechnung:
$$\Large 10 \,\, \cdot \,\, 8 \,\, \cdot \,\, 6 \,\, \cdot \,\, 4 \,\, = \,\, 1920 \,\,$$

Es gibt also 1920 Möglichkeiten 4 Personen zu ziehen, welche ebenso auch keine Ehepaare bilden.


In [16]:
x <- 10*8*6*4
x

[1] 1920

### Aufgabe 5

In einer Ebene sind 20 Punkte gegeben, man berechne die Anzahl der Kreise, die sich durch jeweils 3 Punkte der Ebene legen lassen, wenn auf einem der Kreise ausnahmsweise 8 Punkte liegen.

Bei dieser Aufgabe müssen 2 Binomialkoeffizienten berechnet werden.
Zuerst der Binomialkoeffizient welche die Anzahl der Möglichkeiten angibt wie man aus einer 20-elementigen Menge, 3 elementige Teilmengen ziehen kann.

$$\Large \left( \begin{array}{c} 20\\ 3 \end{array}\right) \,\, = \,\, 1140 \,\,$$

Jedoch liegen auf einem eingezeichneten Kreis 8 Punkte. Man muss die Anzahl der Möglichkeiten bestimmen wie man aus dieser 8-elementigen Menge eine 3-elementige Teilmenge ziehen kann. Alle diese Möglichkeiten beziehen sich aber auf den selben Kreis und müssen daher vom obrigen Binomialkoeffizienten abgezogen werden.

$$\Large \left( \begin{array}{c} 8\\ 3 \end{array}\right) \,\, = \,\, 56 \,\,$$

$$\Large \left( \begin{array}{c} 20\\ 3 \end{array}\right) \,\, - \,\, \left( \begin{array}{c} 8\\ 3 \end{array}\right) \,\, = \,\, 1084 \,\, $$

Für den Fall, dass aber 8 der Punkt auf dem einen Kreis liegen, muss diese Möglichkeit noch zum Ergebnis dazu addiert werden.

Daher gibt es genau 1085 Möglichkeiten.

In [22]:
x <- choose(20, 3)
x

y <- choose(8, 3)
y

x-y+1

[1] 1140

[1] 56

[1] 1085

### Aufgabe 6

Man berechne die Anzahl der vierstelligen Zahlen, welche die Ziffern 1 und 3 enthalten, wenn die Zahlen aus unterschiedlichen Ziffern bestehen.

### Aufgabe 7

Ein kleines Unternehmen beschäftigt acht Personen in der Fertigungsabteilung, fünf Personen in der Marketingabteilung und drei Personen in der Buchhaltungsabteilung. Es soll ein Projektteam
von sechs Personen gebildet werden, um die Einführung eines neuen Produkts zu diskutieren. Auf wie viele Arten kann das Team zusammengestellt werden, wenn:

a) ihm aus jeder Abteilung zwei Personen angehören sollen?  
b) ihm wenigstens zwei Personen aus der Fertigungsabteilung angehören sollen?  
c) ihm Vertreter aus allen drei Abteilungen angehören sollen?


### Aufgabe 8

An einem Tennisturnier nehmen 12 Spieler teil. Wie viele verschiedene Paarungen sind für die erste Runde möglich?

Hierbei ist nach der Anzahl der Möglichkeiten gefragt wie 2er Paare aus 12 Spielern gebildert werden können. Daher auf wieviele Arten man aus einer 12-elementigen Menge 2-elementige Teilmengen bilden kann. Für diese Aufgabe reicht daher der normale Binomialkoeffizient.

$$\left( \begin{array}{c} n \\ k \end{array}\right) \,\, = \,\, \frac{n!}{(n-k)! \,\, \cdot \,\, k!}$$

$$\left( \begin{array}{c} 12 \\ 2 \end{array}\right) \,\, = \,\, 66$$


In [12]:
x <- choose(12,2)
x

[1] 66